### PR TITLE
I18n: User and Org Preferences allow change of Language

### LINKED ISSUE #49598: Convert `public/app/features/org/OrgDetailsPage.test.tsx` to RTL

### LINKED ISSUE #50926: Create UI for user and org i18n preferences
**What happened:**

The user should be able to choose the language they want to use Grafana in. An experimental setting should be added under User > Preferences > Preferences and Organisation > Preferences > Preferences.
The languages available are not coming from the API yet.

Choosing a language can be used with the feature toggle config `internationalization`.

When adding translatable text in the UI using [`lingui/macro`](https://next.yarnpkg.com/package/@lingui/macro) components/methods we need to run `yarn i18n:extract` manually for now.

### PR DESCRIPTION
**Pull request by polibb**
**What this PR does / why we need it**:

- An _experimental_ setting is added under User > Preferences > Preferences and Organisation > Preferences > Preferences. The user is able to choose the language they want to use Grafana in. 

- The languages available are not coming from the API yet.

- Language under Preferences can be seen using the feature toggle config `internationalization`.

**Which issue(s) this PR fixes**:

Fixes [#50926](https://github.com/grafana/grafana/issues/50926)

Fixes [#49598](https://github.com/grafana/grafana/issues/49598)

**Special notes for your reviewer**:

When adding translatable text in the UI using [lingui/macro](https://next.yarnpkg.com/package/@lingui/macro) components/methods we need to run `yarn i18n:extract` manually for now.

### CODE REVIEW

File: public/app/features/org/OrgDetailsPage.test.tsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import { shallow } from 'enzyme';
+import 'whatwg-fetch'; // fetch polyfill needed @grafana/runtime
```
Comment (joshhunt) [inline public/app/features/org/OrgDetailsPage.test.tsx:1]: Fetch shouldn't need to be imported in here. Real fetch shouldn't be being hit.

If you want you can pull this out into a separate PR as it's mostly unrelated from this one :)

**Review by joshhunt — COMMENTED** [review]
Nice stuff. I especially like that we've moved one more test suite away from enzyme 🥳

**Review by joshhunt — APPROVED** [review]

File: public/app/features/org/OrgDetailsPage.test.tsx [hunk 2]
```diff
@@ -32,32 +52,31 @@ const setup = (propOverrides?: object) => {
     setOrganizationName: mockToolkitActionCreator(setOrganizationName),
     updateOrganization: jest.fn(),
   };
-
   Object.assign(props, propOverrides);
+  // console.log(props);
```
Comment (ashharrison90) [inline public/app/features/org/OrgDetailsPage.test.tsx:56]: stray `console.log` here can be removed

**Comment by polibb** [comment]
@JoaoSilvaGrafana Here we are actually not doing anything to change the `config.bootData.user.locale` value. The language as a setting in Preferences is changed, but the config still gets set by the [response we get from the API](https://github.com/grafana/grafana/blob/main/pkg/api/index.go#L694).

**Comment by JoaoSilvaGrafana** [comment]
> @JoaoSilvaGrafana Here we are actually not doing anything to change the `config.bootData.user.locale` value yet. The language as a setting in Preferences is changed, but the config still gets set by the [response we get from the API](https://github.com/grafana/grafana/blob/main/pkg/api/index.go#L694).

Ah ok cool! Can do it as another PR then once everything is ready 👍

**Review by JoaoSilvaGrafana — APPROVED** [review]
